### PR TITLE
RN fixes for devices

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] alignment issues
- [x] add new working

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/devices/render.native.js [hunk 1]
```diff
@@ -86,83 +45,112 @@ const RevokedDescription = () => (
...
+    {title: 'Cancel', onClick: () => {}},
+  ]
+
+  return <Box style={{...stylesCommonRow, alignItems: 'center'}}>
     <Box style={stylesCommonColumn}>
       <Icon type='icon-devices-add-64-x-48' />
     </Box>
     <Box style={stylesCommonColumn}>
-      <Text type='BodyPrimaryLink' onPress={addNewDevice}>Add new...</Text>
+      <Text type='BodyPrimaryLink' onClick={() => ActionSheetIOS.showActionSheetWithOptions({
```
Comment (cjb) [inline shared/devices/render.native.js]: Hm, here's an iOS import in a shared render.native.js -- what does it do on Android?

**Comment by cjb** [comment]
:thumbsup: % C

**Comment by chrisnojima** [comment]
yeah, lemme block this on @awendland 's popup instead

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready again

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**